packages/dev-utils/tsconfig-base.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "rootDir": "src",
+    "declaration": true,
+    "esModuleInterop": true,
+    "types": ["node", "@types/jest", "vitest/globals"],
```

Comment:
is `"vitest/globals"` supposed to be here?

Reply:
i thought we didnt want test types globally in non test

Reply:
oh yeah it doesnt do anything on top of that.